test/TestProgramSettings.cc

```diff
@@ -214,4 +216,42 @@ TEST_F(ProgramSettingsTest, MultipleImagesFromPositional) {
     EXPECT_EQ(settings.files[0], casa_image.string().substr(1));
     EXPECT_EQ(settings.files[1], fits_image.string().substr(1));
     EXPECT_EQ(settings.files[2], hdf5_image.string().substr(1));
+}
+
+TEST_F(ProgramSettingsTest, ExpectedValuesLongJSON) {
+    // auto settings = SettingsFromString(
```

Comment:
can we remove these commented out lines?

Reply:
Done